src/components/SidePanels/DetailsPanel/RegionEditor.tsx

```diff
@@ -73,8 +73,7 @@ const RegionEditorComponent: FC<RegionEditorProps> = ({
           endTime={region.end}
           minTime={0}
           maxTime={region?._ws_region?.duration}
-          startTimeReadonly={true}
-          endTimeReadonly={true}
+          endTimeReadonly={false}
```

Comment:
is there any reason this is set to false? I see the others just removed these properties.

Reply:
Brandon, endTimeReadonly was set to true by default, but it makes more sense for both to be false by default and we set true when we need. I already changed this!